setup.py

```diff
@@ -34,9 +34,6 @@
     classifiers="""\
 License :: OSI Approved :: Zope Public License
 Programming Language :: Python
-Programming Language :: Python :: 2
```

Comment:
Maybe add ``Programming Language :: Python :: 3`` and a line for the actually tested versions here?

Reply:
I thought I did that. I think I screwed up a bit of git here. Good catch.